gcloud_bigtable/row.py

```diff
@@ -400,6 +400,9 @@ def _commit_mutate(self, timeout_seconds=None):
         :param timeout_seconds: Number of seconds for request time-out.
                                 If not passed, defaults to value set on row.
 
+        :type async: bytes
```

Comment:
I haven't before seen this particular documentation markup, but are you saying that the type of the `async` parameter to this function is `bytes`? Why not `bool`?